HTTPRequest/HTTPRequest.Unity.cs

```diff
@@ -137,6 +137,14 @@ private IEnumerator WaitLoop()
 						cookies.SetCookies(requestUri, headerStr.Remove(0, 11));
 					}
 				}
+			} else {
+				UnityEngine.Debug.Log("WWW does not have the responseHeadersString instance, perhaps it is deprecated?");
```

Comment:
maybe this should be a warning? Also please use the MAGE logger otherwise it won't come up on the server or be configurable

Reply:
I don't believe we'd want to send this to servers; this is really something we should see during development, increasing logging costs (since MAGE logs are being stored) wouldn't really add much I think.

So if thats OK with you, I'd switch to Debug.LogWarning instead.

Reply:
hmmmm, sure I guess, though I prefer this to be configurable
for example if we put it on info level using mage logger, we won't send to server, but can see during development. Also during development we can switch off the log level if we don't care (maybe even debug level). BTW UnityEngine logs are stored on the device, this is why we prefer to configure them. Once we turn it off we don't even store junk on the device.

Reply:
Understood, thanks for the clarification. In which case, perhaps using Mage.Logger.Debug would make more sense, to make sure the log isn't forwarded to the server in sanely configured production builds? I am assuming that Info might be forwarded.

Reply:
Production is generally Notice and above, testing is usually info above, development is using info and above unless debug is turned on.